modules/nvidia-gpu-features.adoc

```diff
@@ -0,0 +1,71 @@
+
+NVIDIA Container Toolkit::
+The NVIDIA Container Toolkit allows you to build and run GPU-accelerated containers.
+The toolkit includes a container runtime library and utilities to automatically configure
+containers to leverage NVIDIA GPUs.
+
+Node Feature Discovery (NFD) Operator::
+The Node Feature Discovery Operator (NFD) manages the detection of hardware features and configuration in an OpenShift Container Platform cluster by labeling the nodes with hardware-specific information. NFD labels the host with node-specific attributes, such as PCI cards, kernel, operating system version, and so on.
++
+The NFD Operator can be found on the Operator Hub by searching for “Node Feature Discovery”.
+
+NVIDIA AI Enterprise::
+NVIDIA AI Enterprise is an end-to-end, cloud-native suite of AI and data analytics software, optimized, certified, and supported by NVIDIA with NVIDIA-Certified Systems.
++
+NVIDIA AI Enterprise provides access to a pre-configured GPU Operator within the NVIDIA Enterprise Catalog.
+The GPU Operator is pre-configured to simplify the provisioning experience with NVIDIA AI Enterprise deployments.
++
+NVIDIA AI Enterprise includes support for Red Hat {product-title}. The following methods of installation are supported:
++
+* {product-title} on bare metal or VMware vSphere with GPU Passthrough.
+
+* {product-title} on VMware vSphere with NVIDIA vGPU.
+
+
+Multi-Instance GPU (MIG) Support in {product-title}::
+MIG is useful anytime you have an application that does not require the full power of an entire GPU.
+The new NVIDIA Ampere architecture’s MIG feature allows you to split your hardware resources into multiple
+GPU instances, each exposed to the operating system as an independent CUDA-enabled GPU.
+The NVIDIA GPU Operator version 1.7.0 and above provides MIG feature support for the A100 and A30 Ampere cards.
+These GPU instances are designed to support multiple independent CUDA applications (up to 7), so they operate completely isolated from each other using dedicated hardware resources.
++
+The compute units of the GPU, in addition to its memory, can be partitioned into multiple MIG instances.
+Each of these instances presents as a stand-alone GPU device from the system perspective and can be bound to any application, container, or virtual machine running on the node.
++
+From the perspective of the software consuming the GPU each of these MIG instances looks like its own individual GPU.
+
+GPU Operator dashboard::
+You can install a console plugin to show
+GPU usage information in the cluster utilization screen in the {product-title} web console. GPU usage information includes
+the number of available GPUs, power usage (in watts) for each GPU, and
+the percentage of GPU workload dedicated to video encoding and decoding.
+
+Time-slicing NVIDIA GPUs in OpenShift::
+The latest generations of NVIDIA GPUs provide a mode of operation called Multi-Instance GPU (MIG).
```

Comment:
Looks like this is same as MIG support.  Please refer to this link for a description of GPU time-slicing: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-sharing.html

Reply:
Updated